pkg/telemetry/client.go

```diff
@@ -132,7 +142,24 @@ func (tc *telemetryClient) SaveMetrics() error {
 
 // SendMetrics sends the local stored metrics to super collider
```

Comment:
Could you please add little more doc about how telemetry plugin fetches the metrics db info! (which is from the telemetryOptions in config)

Reply:
Added doc comment